src/components/index.ts

```diff
@@ -0,0 +1,52 @@
+import { ComponentName } from "./component-name/component-name";
```

Comment:
Looks good, but since we have a lot of components, I would use A-Z sorting for not miss/better search